src/main/resources/logback.groovy

```diff
@@ -193,8 +192,7 @@ logger("com.nike.metrics.newrelic.NewRelicReporter", WARN, allAsyncAppendersArra
 
 logger("com.nike.cerberus", INFO, allAsyncAppendersArray, false)
 
-// TODO revert before merge to master
-logger("com.nike.cerberus.jobs.HystrixMetricsProcessingJob", DEBUG, allAsyncAppendersArray, false)
+logger("com.nike.cerberus.jobs.HystrixMetricsProcessingJob", INFO, allAsyncAppendersArray, false)
```

Comment:
What was "revert before merge to master"?

Are there other bad changes in here?

Is this highlander?  Things we need to update?

Reply:
That was referencing the LOG level of DEBUG, I switched it back to INFO